tests/BrowscapUpdaterTest.php

```diff
@@ -76,7 +76,7 @@ public function testSetGetCache()
             ->disableOriginalConstructor()
             ->getMock();
 
-        self::assertSame($this->object, $this->object->setCache($cache));
+        $this->object->setCache($cache);
```

Comment:
Test changes need to be reverted too to ensure we don't break BC accidentally in future :)